core/common/src/main/java/alluxio/util/network/NetworkAddressUtils.java

```diff
@@ -65,11 +65,51 @@ public final class NetworkAddressUtils {
 
   private NetworkAddressUtils() {}
 
+  /**
+   * An interface to get service attributes.
+   */
+  public interface IServiceType {
```

Comment:
nit: `ServiceAttributeProvider`? 

I just never seen `ISomething` in the entire code base.